frontend/src/components/WeekSchedule.js

```diff
@@ -78,8 +79,6 @@ function WeekSchedule({ showModal, courses, hover_course }) {
     let id = 0;
```

Comment:
getting rid of this variable would help keep stable keys, which might also help performance